generic/hwinfo.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Pavithra <pavrampu@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils import distro
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Hwinfo(Test):
+
+
+    is_fail = 0
+
+    def run_cmd(self, cmd):
+        print ("executing ============== %s =================" % cmd)
+        cmd_result = process.run(cmd, ignore_status=True, sudo=True)
```

Comment:
`process.run` returns `output` object. and `process.system_output` returns `output`. I feel better to use `process.system_output`